MCprep_addon/spawner/item.py

```diff
@@ -84,22 +89,21 @@ def reload_items(context):
 		basename = os.path.splitext(os.path.basename(item_file))[0]
 		asset = mcprep_props.item_list.add()
 		asset.name = basename.replace("_", " ")
-		asset.description = "Spawn one " + basename
+		asset.description = f"Spawn one {basename}"
 		asset.path = item_file
 		asset.index = i
 
 		# if available, load the custom icon too
 		if not env.use_icons or env.preview_collections["items"] == "":
 			continue
 		env.preview_collections["items"].load(
-			"item-{}".format(i), item_file, 'IMAGE')
+			f"item-{i}", item_file, 'IMAGE')
 
 	if mcprep_props.item_list_index >= len(mcprep_props.item_list):
 		mcprep_props.item_list_index = len(mcprep_props.item_list) - 1
 
 
-def spawn_item_from_filepath(
-	context, path, max_pixels, thickness, threshold, transparency):
+def spawn_item_from_filepath(context: Context, path: Path, max_pixels: int, thickness: float, threshold: float, transparency: bool) -> Tuple[Optional[Object], Optional[str]]:
```

Comment:
Replace `Object` with `bpy.types.Object`. Normally I would prefer the former, but `Object` can be confused with the `Object` type in Python (I actually had to do a double take on this for this very reason)
```suggestion
def spawn_item_from_filepath(context: Context, path: Path, max_pixels: int, thickness: float, threshold: float, transparency: bool) -> Tuple[Optional[bpy.types.Object], Optional[str]]:
```